commonslibrary/urls.py

```diff
@@ -13,6 +13,7 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^accounts/', include('accounts.urls', namespace='accounts')),
```

Comment:
Remove the namespace here - add it to `accounts/urls.py`, as this generates a warning.